src/bot/commands.rs

```diff
@@ -66,15 +71,23 @@ pub trait Command {
     ) -> String;
 
     fn execute(&self, db_pool: Pool<ConnectionManager<PgConnection>>, api: Api, message: Message) {
-        info!(
-            "{:?} wrote: {}",
-            message.chat.id,
-            message.text.as_ref().unwrap()
-        );
+        let messages = message.text.as_ref().unwrap().to_string();
+
+        let response_message = &messages.replace(BOT_NAME, "");
```

Comment:
Not understand this line of code, why you would like to delete bot name from a message?

Reply:
when we use inline keyboard bot name is added automatically in the text field,so if we send that to the bot commands will not work properly

Reply:
Ok